docs/docs/tutorial/tutorial.md

```diff
@@ -1,7 +1,7 @@
 ---
 layout: page
 title: "Tutorial"
-description: ""
+description: "Tutorial is valid for Spark 1.5"
```

Comment:
good point! I did it here: https://github.com/apache/incubator-zeppelin/commit/2f11ff76044eb6bcb56e7e7b1234df002f2b6056 (hope it is correct since i'm not fluent in git)